openapi/user/time-registration.yaml

```diff
@@ -1,31 +1,31 @@
 get:
   tags:
-  - "Time registration"
+  - "TimeRegistration"
   operationId: "getTimeRegistrationsForUser"
-  description: "Returns time registrations for a user"
+  description: "Returns time registrations that belongs to a user"
   responses:
     "200":
-      description: "List of time registrations for the user"
+      description: "List of time registrations for a user"
```

Comment:
`a` (any) conflicts with the above description (specific):
> Returns time registrations that belongs to a user

So maybe revert this line?

Reply:
will do. Good point